src/resource_server/views/api_v1/classes/cmd.py

```diff
@@ -0,0 +1,40 @@
+import re
+
+from resource_server.utils.commond import paramiko_stablish_connection, read_json_file
+
+
+class CMD:
```

Comment:
similar here ... any further things planned here to make this a useful class? (not just a namespace for static methods)